src/test/java/com/epam/tc/ConfProperties.java

```diff
@@ -0,0 +1,33 @@
+package com.epam.tc;
+
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.util.Properties;
+
+public class ConfProperties {
```

Comment:
It looks like all properties should be removed from `src/test/resources/test.properties`, so this class can be removed, too.

Reply:
I left url, path, key and token in properties, so i do not need to remove this class

Reply:
Are you sure it's needed? As I can see there are no usages of this class anywhere